group.go

```diff
@@ -188,9 +188,12 @@ func (cli *Client) GetGroupRequestParticipants(jid types.JID) ([]types.JID, erro
 		return nil, &ElementMissingError{Tag: "membership_approval_requests", In: "response to group request participants query"}
 	}
 	requestParticipants := request.GetChildrenByTag("membership_approval_request")
-	participants := make([]types.JID, len(requestParticipants))
+	participants := make([]types.GroupParticipantRequest, len(requestParticipants))
 	for i, req := range requestParticipants {
-		participants[i] = req.AttrGetter().JID("jid")
+		participants[i] = types.GroupParticipantRequest{
+			req.AttrGetter().JID("jid"),
+			req.AttrGetter().UnixTime("request_time"),
```

Comment:
```suggestion
			JID: req.AttrGetter().JID("jid"),
			RequestedAt: req.AttrGetter().UnixTime("request_time"),
```

Probably this is what linter is asking you to change

Reply:
Solved! Thank you for your support 😄